manifests/init.pp

```diff
@@ -296,6 +296,11 @@
       remote_user_environ_name => 'HTTP_REMOTE_USER',
       manage_apache            => false,
       servername               => $foreman::servername,
+      postgresql_manage_db     => $foreman::db_manage,
```

Comment:
IMHO these should not be the same ones as Foreman. They're independent. We have separate parameters for candlepin as well.